dataland-internal-storage/src/test/kotlin/db/migration/V8__MigratePercentagesTest.kt

```diff
@@ -7,12 +7,15 @@ class V8__MigratePercentagesTest {
 
     @Test
     fun `test that the eu taxonomy for financials migration script works as expected`() {
+        val dataType = "eutaxonomy-financials"
```

Comment:
The old scripts were touched, since a common function got an additional argument `dataType`, allowing to pass different dataTypes and not only EU Taxo NF as it was before

Reply:
ok